### DISCUSSION TITLE
Microsoft.Maui.Dependencies and Microsoft.Maui.Extensions

### DISCUSSION THREAD

**Discussion by GeorgeLeithead** [CommunityToolkit/Maui#558] [General]
There are 20 warnings when building the solution(s) all stating the following:

"Warning NETSDK1023: A PackageReference for 'Microsoft.Maui.Dependencies' was included in your project. This package is implicitly referenced by the .NET SDK and you do not typically need to reference it from your project."

I have tested removing both packages from all the solutions, building and testing.  All "appears" to be working OK, and this reduces the warnings to only a few API version issues (CA1416).

Should/could these packages be removed?

**Comment by TheCodeTraveler** [comment]
> Should/could these packages be removed?

Nope, we did that on purpose. 

It's a workaround to ensure that the MAUI dependencies remain private and do not become dependencies on the NuGet Package. 

It was a bug in the initial release of MAUI, which we still target. 

We'll likely remove these references when we bump our minimum dependency to .NET 7 in a few months.

**Reply by GeorgeLeithead** [reply]
Remarkably interesting and enlightening.  Thanks for the explanation.

**Reply by MisinformedDNA** [reply]
> > Should/could these packages be removed?
> 
> Nope, we did that on purpose.
> 
> We'll likely remove these references when we bump our minimum dependency to .NET 7 in a few month

Should they be removed? "No. We will remove them." lol

**Reply by GeorgeLeithead** [reply]
I believe that the minimum dependency is now .NET 7.  So have these been removed already, or are being removed? @brminnick

**Reply by TheCodeTraveler** [reply]
Yup. This Discussion is obsolete now that we target .NET 7.
